documentation/api.md

```diff
@@ -76,3 +76,15 @@ CRUD for Visual Rules. Originally detailed in [DCA documentation](https://colabo
 | ------ | :----------------------- | ----------------------------------- |
```

Comment:
Related issue #364 already cited in CNR?

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/396/commits/fe4fda790b23f1c76c75ebd65cbcaff5ecb9236d